core/src/test/scala/io/snappydata/app/CreateIndexTest.scala

```diff
@@ -16,33 +16,104 @@
  */
 package io.snappydata.app
```

Comment:
right, I will be putting in dunits but before that have lots of outstanding items still to cover. for e.g. I am in the process of putting in replicated region and 3 or more N-way join index pickup.